tests/unit/amo/components/TestEditableCollectionAddon.js

```diff
@@ -76,4 +83,164 @@ describe(__filename, () => {
     sinon.assert.called(clickEvent.stopPropagation);
     sinon.assert.calledWith(removeAddon, addon.id);
   });
+
+  describe('notes area', () => {
+    it('hides the notes area by default', () => {
+      const root = render();
+
+      expect(root.find('.EditableCollectionAddon-notes')).toHaveLength(0);
+    });
+
+    it('shows the read-only version of the notes area if there are notes', () => {
+      const notes = 'Some notes.';
+      const addon = {
+        ...fakeAddon,
+        notes,
+      };
+      const root = render({ addon });
+      const notesArea = root.find('.EditableCollectionAddon-notes');
+      expect(notesArea.find(Icon)).toHaveProp('name', 'comments-blue');
+      expect(root.find('.EditableCollectionAddon-notes-read-only'))
+        .toHaveLength(1);
+      expect(notesArea.find('.EditableCollectionAddon-notes-content'))
+        .toHaveProp('dangerouslySetInnerHTML', sanitizeHTML(notes));
+      const editButton = notesArea.find(Button);
+      expect(editButton).toHaveClassName('EditableCollectionAddon-notes-edit-button');
+      expect(editButton).toHaveProp('buttonType', 'action');
+      expect(editButton).toHaveProp('micro', true);
+      expect(editButton).toHaveProp('onClick', root.instance().onEditNote);
+      expect(editButton).toHaveProp('buttonType', 'action');
+      // The form should not be shown.
+      expect(root.find('.EditableCollectionAddon-notes-form')).toHaveLength(0);
+    });
+
+    it('shows an empty notes form when the comment icon is clicked', () => {
+      const root = render();
+
+      expect(root.find('.EditableCollectionAddon-notes')).toHaveLength(0);
+      const commentIcon = root.find('.EditableCollectionAddon-create-note');
+      const clickEvent = createFakeEvent();
```

Comment:
I don't think we are using this variable more than once, so I'd suggest to not create it.